graphql-server/go-server/graph/schema/worker.gql

```diff
@@ -126,6 +129,7 @@ mutation updateWorker($input: UpdateWorkerInput) {
             memory
             nvidiaGPU
           }
+          api
```

Comment:
so all workers will have the same api address? as we just use ExternalAPIServer from config file.

Reply:
Yes